pylint/checkers/exceptions.py

```diff
@@ -279,14 +295,15 @@ def open(self):
         "notimplemented-raised",
         "bad-exception-context",
         "raising-format-tuple",
+        "raise-missing-from",
     )
     def visit_raise(self, node):
         if node.exc is None:
             self._check_misplaced_bare_raise(node)
             return
 
-        if node.cause:
```

Comment:
I think it makes more sense to do :
```
if node.cause:
    self._check_bad_exception_context(node)
else:
    self._check_raise_missing_from(node)
```
Then remove the ```  if not node.cause:            return``` and ` if node.cause is not None: return` in the two functions.

Reply:
Agreed and fixed.